mlflow/utils/rest_utils.py

```diff
@@ -350,9 +350,6 @@ def get_set_trace_tag_endpoint(request_id):
 
 
 def call_endpoint(host_creds, endpoint, method, json_body, response_proto, extra_headers=None):
-    # Convert json string to json dictionary, to pass to requests
-    if json_body:
-        json_body = json.loads(json_body)
```

Comment:
We seem to be assuming that `json_body` is only specified by one particular upstream method. That might not be the case. This could break other callers, right?